packages/app/src/cli/services/build/extension.ts

```diff
@@ -175,6 +175,12 @@ export async function buildFunctionExtension(
     } else {
       await buildOtherFunction(extension, options)
     }
+
+    const wasmOpt = (extension as ExtensionInstance<FunctionConfigType>).configuration.build.wasm_opt
+    if (fileExistsSync(extension.outputPath) && wasmOpt) {
+      await runWasmOpt(extension.outputPath)
```

Comment:
Is this a slow process (esp. on first download)? Is there some user feedback on what's happening here? I'm not clear if this method is contained in something that provides feedback, or if `runWasmOpt` would need to provide its own.

Reply:
`outputDebug` will only be included with `--verbose` though right?

Reply:
Nick's correct about what outputDebug does.

I see this is merged now, but yes I think you should put some feedback in place. Its just about avoiding the perception of things being frozen. Whether thats just the download or its the download & optimisation I think is a call or you and team.